Util/Factory/Filter/DatatableFilter.php

```diff
@@ -74,20 +77,41 @@ public function getFilterValues()
         return $this->filter_values;
     }
 
+    /**
+     * @return mixed|null
+     */
+    public function getDefaultValue()
+    {
+        return $this->default_value;
+    }
+
+    /**
+     * @param mixed|null $default_value
+     * @return DatatableFilter
+     */
+    public function setDefaultValue($default_value)
+    {
+        $this->default_value = $default_value;
+        return $this;
+    }
+
     /**
      * Static helper to easily create boolean filter
      *
      * @param string $yes_label
      * @param string $no_label
+     * @param bool|null $default_value
      * @return DatatableFilter
      */
-    public static function constructBooleanFilter($yes_label = 'yes', $no_label = 'no')
+    public static function constructBooleanFilter($yes_label = 'yes', $no_label = 'no', $default_value = null)
     {
         return new self(
             array(
                 new DatatableFilterValue(1, $yes_label),
                 new DatatableFilterValue(0, $no_label)
-            )
+            ),
+            self::SEARCH_TYPE_EQUALS,
+            (int)$default_value
```

Comment:
$default_value is now always `0` since `null` get's cast to `0` using `(int)null`